transformers/integration_tests/median_lift_test.go

```diff
@@ -0,0 +1,87 @@
+package integration_tests
+
+import (
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lib/pq"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/median_lift"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MedianLift EventTransformer", func() {
+	medianLiftConfig := event.TransformerConfig{
+		TransformerName:   constants.MedianLiftTable,
+		ContractAddresses: test_data.MedianAddresses(),
+		ContractAbi:       constants.MedianABI(),
+		Topic:             constants.MedianLiftSignature(),
+	}
+
+	It("transforms Median Lift log events", func() {
+		blockNumber := int64(8936530)
+		medianLiftConfig.StartingBlockNumber = blockNumber
+		medianLiftConfig.EndingBlockNumber = blockNumber
+
+		test_config.CleanTestDB(db)
+
+		header, headerErr := persistHeader(db, blockNumber, blockChain)
+		Expect(headerErr).NotTo(HaveOccurred())
+
+		// TODO: fetch event from blockchain once one exists
+		// logFetcher := fetcher.NewLogFetcher(blockChain)
+		// logs, logErr := logFetcher.FetchLogs(
+		// 	event.HexStringsToAddresses(medianLiftConfig.ContractAddresses),
+		// 	[]common.Hash{common.HexToHash(medianLiftConfig.Topic)},
+		// 	header)
+		// Expect(logErr).NotTo(HaveOccurred())
+
+		eventLogs := test_data.CreateLogs(header.Id, []types.Log{test_data.RawMedianLiftLogWithFiveAccounts}, db)
+
+		transformer := event.ConfiguredTransformer{
+			Config:      medianLiftConfig,
+			Transformer: median_lift.Transformer{},
+		}.NewTransformer(db)
+
+		transformErr := transformer.Execute(eventLogs)
+		Expect(transformErr).NotTo(HaveOccurred())
+
+		var dbResults []MedianLiftModel
+		err := db.Select(&dbResults, `SELECT address_id, msg_sender, a_length FROM maker.median_lift ORDER BY id`)
```

Comment:
What is the `ORDER BY` accomplishing here?

Reply:
Err.. nothing.